client/domain/TaskObjectFactorySpec.js

```diff
@@ -0,0 +1,65 @@
+// Copyright 2017 The TIE Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Unit tests for TaskObject domain objects.
+ */
+
+describe('TaskObjectFactory', function() {
+  var QuestionObjectFactory;
+  var question;
+
+  var TITLE = 'title';
```

Comment:
You don't need to pull these out as constants. Just include them within the definition of `question` below.